How-to-reset-qBittorrent-webui-password.md

```diff
@@ -0,0 +1,48 @@
+# How to reset qBittorrent webui password
+
+1. Check the version of qBittorrent
+
+2. Stop qBittorrent
+
+3. Open qBittorrent configuration file
+
+- Windows `%APPDATA%\qBittorrent\qbittorrent.ini`
+- Linux `~/.config/qBittorrent/qBittorrent.conf`
+- macOS `~/.config/qBittorrent/qBittorrent.conf`
+
+4. Remove the password line from the configuration file
+
+- **qBittorrent version < 4.2.0**, password line start with `WebUI\Password_ha1`, remove it
+
+example
+
+```conf
+WebUI\Password_ha1=@ByteArray(f6fdffe48c908deb0f4c3bd36c032e72)
+```
+
+- **4.2.0 <= qBittorrent version <= 4.6.0**, password line start with `WebUI\Password_PBKDF2=`, remove it
+
+example
+
+```conf
+WebUI\Password_PBKDF2="@ByteArray(kN0rB3p5kk4yo3u7hzK9Ag==:n3ebbj5tzAf8Gkruib2lgUwDIO/ur7gdDEOKIh+B6PC1IEiyJiq536pxB3scDTkDJTBwn2j4AvLAozEBaYsvBw==)"
+```
+
+- **4.6.0 < qBittorrent version**, password line start with `WebUI\Password_PBKDF2=`, replace with the following line
```

Comment:
Seems this doesn't fit in `4. Remove the password line from the configuration file`.

Reply:
`4. Remove the password line from the configuration file` -> `4. Remove or replace the password line from the configuration file`

Reply:
In qBittorrent v5.0+ you can also just remove existing line in order to reset current password.